packages/panzer/disc-fe/src/Panzer_FaceToElement_impl.hpp

```diff
@@ -100,7 +101,7 @@ void
 FaceToElement<LocalOrdinal,GlobalOrdinal>::
 initialize(panzer::ConnManager & conn)
 {
-  Teuchos::RCP<const Teuchos::Comm<int>> comm_world(new Teuchos::MpiComm< int>(MPI_COMM_WORLD));
+  Teuchos::RCP<const Teuchos::Comm<int>> comm_world(new Teuchos::MpiComm< int>(panzer::get_global_comm()));
```

Comment:
This is not the correct fix. There is a method just above that takes a `comm` argument. Just allow `MPI_COMM_WORLD` here.

Reply:
Fixed